### ISSUE TITLE
Training details of RGB tokenizer 

### ISSUE THREAD

**Issue by Cerf-Volant425** [apple/ml-4m#13]
Thanks for your great work!

As you mentioned in your paper 4M, you used DiVAE to train the RGB tokenizer, first on 100 epochs of ImageNet-1K, and then for an additional 15 epochs on the CC12M dataset, I followed your training settings indicated by your paper, and used the ckpt-100 training on ImageNet-1K as full_ckpt, but I encoutered NaN loss problem when I continued training on the CC12M dataset, could you please provide some suggestions to resolve the problem? 

Thanks in advance!
